### PR TITLE
feat: New parser for systemd_perms

### PR DESCRIPTION
**Pull request by TZ3070**
Signed-off-by: Xinting Li <xintli@redhat.com>

### All Pull Requests:

Check all that apply:

* [x] Have you followed the guidelines in our Contributing document, including the instructions about commit messages?
* [ ] Is this PR to correct an issue?
* [ ] Is this PR an enhancement?

### Complete Description of Additions/Changes:

*Add your description here*

### CODE REVIEW

**Comment by TZ3070** [comment]
@xiangce Please have a review.

**Review by xiangce — CHANGES_REQUESTED** [review]
@TZ3070 - These two parsers are in fact `Ls` like parsers, would you please rename them with a prefix `Ls` like others, e.g. 
https://github.com/RedHatInsights/insights-core/blob/cc2e26225b1db85eb11e3b160cb12b81d1b3796c/insights/parsers/ls_sys_firmware.py#L45-L46

File: insights/parsers/ls_systemd.py
Comment (xiangce) [inline insights/parsers/ls_systemd.py:7]: This should be modified too.

File: insights/parsers/ls_systemd.py
Comment (xiangce) [inline insights/parsers/ls_systemd.py:10]: The same comment as above

File: insights/parsers/ls_systemd.py
Comment (xiangce) [inline insights/parsers/ls_systemd.py:21]: How above remove the part of `PermsParser`?

File: insights/parsers/ls_systemd.py
Comment (xiangce) [inline insights/parsers/ls_systemd.py:69]: How above remove the part of `PermsParser`?

**Review by xiangce — CHANGES_REQUESTED** [review]
@TZ3070 - Please check my new comments above

And 1) since this is a new parser file, please add doc entry for it in the `docs/shared_parsers_catalog/` dir, so that its online document can be generated automatically. 2), for newly added specs parsers, please raise an MR to the insights-core-assets repo at first to make sure they can be added to the uploader.json.

File: insights/specs/default.py
Comment (xiangce) [inline insights/specs/default.py:374]: You also need to add the `simple_command` spec to the `insights_archive.py` as a `simple_file` to make sure the `core` could find it from the insights-archive successfully. 

Please use the `mangle` command provide by the insights-core to get the `simple_file` of this `simple_command`

File: insights/specs/default.py
Comment (xiangce) [inline insights/specs/default.py:386]: Same as above

**Review by xiangce — APPROVED** [review]

**Review by xiangce — APPROVED** [review]
Approved pending the approval of Specs.

**Pull request merged**
